test/e2e/app_management_test.go

```diff
@@ -373,6 +373,49 @@ func TestKsonnetApp(t *testing.T) {
 		})
 }
 
+func TestKustomize2AppSource(t *testing.T) {
```

Comment:
Can you move this to a file name `kustomize_test.go` so we have all Kustomize e2e tests all in one file?

Reply:
Ah... didn't realize that. Will do.